scylladb/scylla
# docs: moved latest_version to conf.py

**Related issues:** https://github.com/scylladb/sphinx-scylladb-theme/issues/87

All the variables related to the multiversion extension are now defined in ``conf.py`` instead of using the GitHub Actions file.

##  How to test this PR

Run ``make multiversionpreview`` on ``docs`` folder. When you open https://0.0.0.0:5500,  the browser should render the documentation site.
